src/components/sub-navigation/SubNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { HTMLProps } from 'react';
+
+interface SubNavigationItemProps extends HTMLProps<HTMLLIElement> {
+  text: string,
+  active?: boolean,
+  target: string
+}
+
+interface SubNavigationPanel extends React.FC<HTMLProps<HTMLDivElement>> {
+  SubNavigationItem: React.FC<SubNavigationItemProps>
+}
+
+const SubNavigation: SubNavigationPanel = ({
+  children,
+  ...rest
+}) => (
+  <nav className="nhsuk-sub-navigation" {...rest}>
+    <ul className="nhsuk-sub-navigation__list">
+      {children}
+    </ul>
+  </nav>
+);
+
+const SubNavigationItem: React.FC<SubNavigationItemProps> = ({
```

Comment:
All props passed to the SubNavigationItem component can be satisfied by a standard HTML Anchor element, i.e. the `text` prop has the same purpose as `children`.